bindsnet/network/__init__.py

```diff
@@ -276,6 +277,12 @@ def run(self, inpts: Dict[str, torch.Tensor], time: int, **kwargs) -> None:
         for c in self.connections:
             self.connections[c].normalize()
 
+        # Decay noise_std for noisy layer.
```

Comment:
This check slows down all networks that don't use noise_decay. Maybe make another Nodes subclass specifically for noise_decay and do the update there? I'm not sure what this is supposed to do

Reply:
Yeah, this seems like feature bloat. Subclassing `Nodes` is a good idea.